validator/test/fixtures/assets/service_accounts/data.json

```diff
@@ -0,0 +1,59 @@
+[
+    {
+        "asset_type": "google.iam.ServiceAccount",
```

Comment:
the asset type doesn't seem to match "iam.googleapis.com/ServiceAccount"?

Reply:
I thought it was "iam.googleapis.com/ServiceAccount" as well, but after looking into the CAI export, it turned out to be "google.iam.ServiceAccount".

Reply:
That's probably done with CAI v1beta1? Let's stick with the asset type used in CAI v1.

Reply:
changed accordingly.